cfme/tests/automate/custom_button/test_cloud_objects.py

```diff
@@ -44,9 +44,14 @@ def setup_objs(button_group, provider):
     obj_type = button_group[1]
 
     if obj_type == "PROVIDER":
-        # To-Do: Add check for block and object storage manager
+        # Note: For the custom button provider object points
+        # provider, storage managers, network managers
+        bm_coll = provider.appliance.collections.block_managers.filter({"provider": provider})
+        block_manager = bm_coll.all()[0]
+        om_coll = provider.appliance.collections.block_managers.filter({"provider": provider})
```

Comment:
om_coll = provider.appliance.collections.block_managers.filter({"provider": provider})
Shouldn't this be using object_manager instead of block_manager ?

Reply:
aahhh my bad :+1: